src/main/kotlin/com/rootsid/wal/library/mongoimpl/document/BlockchainTxLogDocument.kt

```diff
@@ -11,9 +11,9 @@ data class BlockchainTxLogDocument(
     override val walletId: String,
     override val action: BlockchainTxAction,
     override var description: String? = null,
+    override val createdAt: String,
```

Comment:
I suggest using LocalDateTime
```suggestion
    @Serializable(with = LocalDateTimeSerializer::class) override val createdAt: LocalDateTime = LocalDateTime.now(),
    @Serializable(with = LocalDateTimeSerializer::class) override val updatedAt: LocalDateTime = LocalDateTime.now()
```

Reply:
ok